controllers/dashboardController.js

```diff
@@ -2,6 +2,11 @@ const cookie = require('../utils/cookie')
 const dbAppointments = require('../db/appointments')
 const dbPatient = require('../db/patients')
 
+function hasFolderCompleted(patient) {
+  return (patient.INE !== '') && patient.hasPrescription && patient.isStudentStatusVerified &&
```

Comment:
on peut pas avoir INE === undefined ou null ? 
Et en dessous, doctorName et institutionname undefined ?

Comme c'est des strings, on peut faire juste `!patient.INE`, ca couvre null, undefined, "" (source : https://stackoverflow.com/questions/5515310/is-there-a-standard-function-to-check-for-null-undefined-or-blank-variables-in)

Reply:
Et tu check pas doctorAdress et doctorPhone ? Ca parait difficile à comprendre quels champs sont indispensables ou pas, j'ai peur que ca soit confusing : ya des champs required, des champs pas-required-tu-peux-les-mettre-plus-tard, et des champs en-fait-on-s'en-fout :D

Si ya des champs dont on se fout, il faut carrement ne pas les demander, je pense aussi qu'ils sont un peu useless...

Reply:
> Et tu check pas doctorAdress et doctorPhone ?
j'ai fait remonter la question :slightly_smiling_face:

Reply:
le téléphone du docteur a disparu, et l'adresse correspond à la ville et/ou code postal :heavy_check_mark: